ATLEDGE-553 Modify library examples header info

eriknyquist opened the pull request:
@bigdinotech @calvinatintel @SidLeung ready for review, please read summary below and advise.

Summary of changes; This time, I've made the change only on example files provided by us (instead of the whole corelibs), and I've added all of the example files (4 of them were missed last time), with the following exception;

None of the files in libraries/CurieTimerOne/examples have a proper license. I'm assuming they should have the same license s the other example files-- I'll add it, unless there's a special reason for this I don't know about?

calvinatintel commented on libraries/CurieIMU/examples/TapDoubleDetect/TapDoubleDetect.ino:
```diff
@@ -1,20 +1,6 @@
 /*
    Copyright (c) 2015 Intel Corporation.  All rights reserved.
```
Should be 2016

calvinatintel commented on libraries/CurieIMU/examples/TapDoubleDetect/TapDoubleDetect.ino:
```diff
@@ -67,4 +53,23 @@ static void eventCallback()
+}
+
+/*
+   Copyright (c) 2015 Intel Corporation.  All rights reserved.
```
2016

calvinatintel commented:
Not sure why the CurieTimer files don't have the licenses. @bigdinotech @SidLeung thoughts?

eriknyquist commented:
@calvinatintel  I'll update the dates

SidLeung commented:
I know why there are no license headers in the CurieTimerOne Sketches...  They are great examples of what not to do when one is a newbie and still wants to keep one's job.  Would you be kind enough to fix that for me, @eriknyquist ?

eriknyquist commented:
@SidLeung yes, I fixed Calvin's comment, added licences to the TimerOne files, and also to the CurieEEPROM files which were also missed. I'll push it tomorrow morning.

eriknyquist commented:
@calvinatintel @SidLeung @bigdinotech changes mentioned in previous comment have been amended to the original commit.

calvinatintel commented:
Wait, CurieTimerOne files aren't owned by Intel. Let me examine those a bit more.

calvinatintel commented:
We don't own these files, so we can't put our own license. Let's revert.

- https://github.com/01org/corelibs-arduino101/blob/d4bf00f84b0245f9c9061271f407a1580a73851e/libraries/CurieEEPROM/examples/eeprom_crc/eeprom_crc.ino
- https://github.com/01org/corelibs-arduino101/blob/d4bf00f84b0245f9c9061271f407a1580a73851e/libraries/CurieEEPROM/examples/eeprom_get/eeprom_get.ino
- https://github.com/01org/corelibs-arduino101/blob/d4bf00f84b0245f9c9061271f407a1580a73851e/libraries/CurieEEPROM/examples/eeprom_put/eeprom_put.ino
- https://github.com/01org/corelibs-arduino101/blob/d4bf00f84b0245f9c9061271f407a1580a73851e/libraries/CurieIMU/examples/RawImuDataSerial/RawImuDataSerial.ino
- https://github.com/01org/corelibs-arduino101/blob/d4bf00f84b0245f9c9061271f407a1580a73851e/libraries/CurieTimerOne/examples/CurieTimer1Interrupt/CurieTimer1Interrupt.ino

@bigdinotech , are these files originated from the default Arduino library? We prob also can't put our license on it either.

- https://github.com/01org/corelibs-arduino101/blob/d4bf00f84b0245f9c9061271f407a1580a73851e/libraries/CurieEEPROM/examples/eeprom_read/eeprom_read.ino
- https://github.com/01org/corelibs-arduino101/blob/d4bf00f84b0245f9c9061271f407a1580a73851e/libraries/CurieEEPROM/examples/eeprom_write/eeprom_write.ino

eriknyquist commented:
@calvinatintel commit updated with the changes we discussed (removed all timerone examples, all eeprom example and rawImuSerialData IMU example).

calvinatintel commented:
@eriknyquist The modifications look good. I've only looked over the modified files, so I didn't check whether all examples were looked at. I trust that you've gone through them all?

eriknyquist commented:
@calvinatintel Yes, I have.
